core/asset/discovery.go

```diff
@@ -53,6 +60,9 @@ type SearchConfig struct {
 
 	// Queries is a param to search a resource based on asset's fields
 	Queries map[string]string
+
+	//SearchFlags flags to control the search behavior (e.g. column level search, disable fuzzy, etc)
+	SearchFlags *SearchFlags
```

Comment:
```go
	// Flags to control the search behavior (e.g. column level search, disable fuzzy, etc)
	Flags SearchFlags
```

Don't see any need to use a pointer here. Will simplify usage by not needing `nil` checks.

Reply:
Done